src/RpcClient.ts

```diff
@@ -550,13 +550,22 @@ export class RpcClient {
 
       versionedTransaction = new VersionedTransaction(v0Message);
       versionedTransaction.sign(signers);
+
+      if (feePayer) {
+        versionedTransaction.sign([feePayer]);
```

Comment:
Wouldn't this remove the signatures from the last `sign` call?

Reply:
Since Versioned Transactions don't have a partial sign, should we add `feePayer` to the `signers` array, if it exists?

Reply:
@tavindev — I've added [this commit](https://github.com/helius-labs/helius-sdk/pull/100/commits/f4bbdb7515fab36d40211939d787b25bfb1063e5) which should handle the signers correctly

Reply:
Great!